Makefile

```diff
@@ -20,6 +20,7 @@
 FLAVORS ?= \
 	luminous,opensuse,42.3 \
 	luminous,centos,7 \
+	luminous,debian,stretch-backports \
```

Comment:
is stretch-backports a version of debian ? hum.
The last digit is usually the number of the os release.

Reply:
Stretch is name of Debian 9 release. And to be able to build ceph containers we need some packages from backports repository.
Using 'stretch-backports' as distro release name makes it clean what is used to achieve our goal.

We may use '9' or 'stretch' here but then in PREINSTALL we will have to add backports repo anyway.

Reply:
So looks fine !